functions/views/prescription-template.ejs

```diff
@@ -61,13 +61,15 @@
 			         <p><span><%= details.doctor.name %></span></p>
 			         <p><%= details.doctor.designation %></p>
 			         <p>Medical Registration Number: <%= details.doctor.registration_number %></p>
-			         <p><%= details.doctor.location %></p>
+                     <p><%= details.doctor.location %></p>
+                     <p><%= details.doctor.emailid %></p>
```

Comment:
Use `email` as a key for representing email since you used `email` for patient details. Also update the sendEmail function accordingly.

Reply:
I don't have an API key for airtable, so can't run index.js locally, and won't be able to test the changes in sendEmail function, that's why I haven't changed that part.

Reply:
@shubhamjain2998 Use `apprfEpTNmqA7S2vx` as the base id. This base is similar to the original base, but with no records. I sent an invite to this base to `jains1801@gmail.com`. 

You can go to https://airtable.com/account for `api_key`